app/src/androidTest/java/org/mozilla/focus/screengrab/BrowserFragmentScreenshot.kt

```diff
@@ -126,7 +126,7 @@ class BrowserFragmentScreenshot : BaseScreenshot() {
             loadPageFromHomeSearchField(activityTestRule.activity, TARGET_URL_LICENSE)
             longClickOnWebViewContent(activityTestRule.activity)
             clickTextActionMore()
-            checkSearchInRocketIsDisplayed()
+//            checkSearchInRocketIsDisplayed()
```

Comment:
If you really want to keep this, please state why it's important so in the future other people can remember.
But I prefer to remove it since: https://en.wikipedia.org/wiki/You_aren%27t_gonna_need_it